airflow/www/static/js/utils/colors.json

```diff
@@ -0,0 +1,842 @@
+[
```

Comment:
I don't like having to add an 800 json file to do color mapping just for the note linear-gradient. I wish we could do without it. Or implement a solution that leverage ChakraUI themes and color palette. (We could for instance only map default colors to chakra color palette and then play with the numbers to get brighter colors -> red.400, red.200 etc.)

Reply:
Status colors are not defined in the UI and are stored as strings "green" instead of rgb or hex. I think it would be best to maintain a single source of truth for state colors. I wanted a way to convert it. But using [an existing library](https://www.npmjs.com/package/color) is probably better.

Reply:
Very interesting. Maybe for later, I can do that.